portality/templates/_formhelpers.html

```diff
@@ -219,11 +219,11 @@
             <div class="col-md-9">
               {{ subfield(**kwargs) }}
               {% if subfield.errors %}
-              <ul class="errors">
                   {% for error in subfield.errors %}
-                  <li>{{ error }}</li>
+                      <ul class="errors">
+                        <li>{{ error }}</li>
+                      </ul>
```

Comment:
This seems odd, is that right?  It looks like for each error you'll end up with an unordered list, containing a single list item?

Reply:
My mistake. Fixed.